### PR TITLE
Change term claim to attributes

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: apps/admin-portal/src/components/claims/wizard/external-dialect/external-claims-add-dialect.tsx [hunk 1]
```diff
@@ -107,8 +107,9 @@ export const ExternalClaims = (props: ExternalClaimsPropsInterface): ReactElemen
                             )
                             : (
                                 <EmptyPlaceholder
-                                    title="No External Claim"
-                                    subtitle={ [ "Currently, there is no external claim available for this dialect." ] }
+                                    title="No External Attribute"
+                                    subtitle={ [ "Currently, there is no external attribute available for " + 
+                                    "this dialect." ] }
```
Comment (jeradrutnam) [inline apps/admin-portal/src/components/claims/wizard/external-dialect/external-claims-add-dialect.tsx:112]: This has to be Attributes right ?

File: apps/admin-portal/src/components/claims/add/add-external-claim.tsx [hunk 2]
```diff
@@ -171,10 +171,10 @@ export const AddExternalClaims = (props: AddExternalClaimsPropsInterface): React
                     <Grid.Column width={ 7 }>
                         <Field
                             name="claimURI"
-                            label="Claim URI"
+                            label="Attribute URI"
                             required={ true }
-                            requiredErrorMessage="Claim URI is required"
-                            placeholder="Enter a claim URI"
+                            requiredErrorMessage="Attribute URI is required"
+                            placeholder="Enter a attribute URI"
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/components/claims/add/add-external-claim.tsx:177]:
```suggestion
                            placeholder="Enter an attribute URI"
```

File: apps/admin-portal/src/components/claims/wizard/external-dialect/external-claims-add-dialect.tsx (hunk 1 above)
Comment (DimalChandrasiri) [reply apps/admin-portal/src/components/claims/wizard/external-dialect/external-claims-add-dialect.tsx:112]: Ah yes it's referring to multiple items. So should be `Attributes`.

Comment (DimalChandrasiri) [reply apps/admin-portal/src/components/claims/wizard/external-dialect/external-claims-add-dialect.tsx]: Fixed with commit aeae82d

File: apps/admin-portal/src/pages/local-claims.tsx [hunk 3]
```diff
@@ -217,62 +217,66 @@ export const LocalClaimsPage = (): ReactElement => {
                     : null
             }
             <PageLayout
-                title="Local Claims"
-                description="View, edit and add local claims"
+                title="Local Attributes"
+                description="View, edit and add local attributes"
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/pages/local-claims.tsx:221]:
```suggestion
                description="Create and manage local attributes"
```

File: apps/admin-portal/src/components/claims/wizard/local-claim/basic-details-local-claims.tsx [hunk 4]
```diff
@@ -125,10 +125,10 @@ export const BasicDetailsLocalClaims = (props: BasicDetailsLocalClaimsPropsInter
                         <Field
                             type="text"
                             name="claimURI"
-                            label="Claim ID"
+                            label="Attribute ID"
                             required={ true }
-                            requiredErrorMessage="Claim ID is required"
-                            placeholder="Enter a claim ID"
+                            requiredErrorMessage="Attribute ID is required"
+                            placeholder="Enter a attribute ID"
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/components/claims/wizard/local-claim/basic-details-local-claims.tsx:131]:
```suggestion
                            placeholder="Enter an attribute ID"
```

**Pull request merged**